packages/storybook/src/nimble/select/select-opened-matrix.stories.ts

```diff
@@ -174,3 +211,48 @@ export const selectBelowOpenNoFilterDarkThemeWhiteBackgroundWithPlaceholder: Sto
     component(positionStates[0], FilterMode.none, true),
     darkThemeBlackBackground
 );
+
+export const selectGroupedOptionsLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true),
+    lightThemeWhiteBackground
+);
+
+export const selectGroupedOptionsColorThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true),
+    colorThemeDarkGreenBackground
+);
+
+export const selectGroupedOptionsDarkThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true),
+    darkThemeBlackBackground
+);
+
+export const selectGroupedAndNotGroupedOptionsLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true, true),
+    lightThemeWhiteBackground
+);
+
+export const selectGroupedAndNotGroupedOptionsColorThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true, true),
+    colorThemeDarkGreenBackground
+);
+
+export const selectGroupedAndNotGroupedOptionsDarkThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true, true),
+    darkThemeBlackBackground
+);
+
+export const selectGroupedWithSlottedLabelLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true, false, true),
+    lightThemeWhiteBackground
+);
+
+export const selectGroupedWithSlottedLabelColorThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true, false, true),
+    colorThemeDarkGreenBackground
+);
+
+export const selectGroupedWithSlottedLabelDarkThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    component(positionStates[0], FilterMode.standard, true, true, false, true),
+    darkThemeBlackBackground
```

Comment:
The names of these stories don't match the names of the variables being passed for the theme.